### PR TITLE
Errata update for 4.5.32 release

### PR DESCRIPTION
**Pull request by codyhoag**
Ships 2021-02-16

### CODE REVIEW

**Comment by codyhoag** [comment]
Preview: http://file.rdu.redhat.com/~choag/021121/RNs-4-5-32/release_notes/ocp-4-5-release-notes.html#ocp-4-5-32

File: release_notes/ocp-4-5-release-notes.adoc [hunk 1]
```diff
@@ -2504,3 +2504,48 @@ link:https://access.redhat.com/solutions/5780621[{product-title} 4.5.31 containe
...
+[id="ocp-4-5-32-features"]
+==== Features
+
+[id="ocp-4-5-32-enhancements-insights-operator"]
+===== Enhancements to Insights Operator
+
+The Insights Operator now collects the following additional information:
+
+* Config files for `MachineConfigPool` objects
+* The number of install plans
```
Comment (ahardin-rh) [inline release_notes/ocp-4-5-release-notes.adoc:2528]: s/install/installation (unless install plans are a common term)

File: release_notes/ocp-4-5-release-notes.adoc [hunk 2]
```diff
@@ -2504,3 +2504,48 @@ link:https://access.redhat.com/solutions/5780621[{product-title} 4.5.31 containe
...
+
+* Previously, a misplaced array index was causing issues while pruning records from memory. This caused excessive memory usage and kept old reports in the archive. This was fixed by changing the array index key. Now pruning removes records from memory, avoiding memory overflow and keeping records updated in the archive. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894243[*BZ#1894243*])
+
+* Previously, attempts to write to a nil writer could have led to `invalid memory address` or `nil pointer dereference` errors. Also, sharing the instance of the writer could have led to `index out of range [43] with length 30 and recovered from err index > windowEnd` errors. This has been fixed in the `SerializeObject` function in the Kube API server. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1879208[*BZ#1879208*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-5-release-notes.adoc:2546]: s/Kube/Kubernetes

**Comment by ahardin-rh** [comment]
Just 2 minor suggestions. Otherwise, looks great!

**Comment by codyhoag** [comment]
Closing, as this was dropped and has been resent for 4.5.33: #29848

**Pull request closed without merging**